e2e/helpers.ts

```diff
@@ -88,7 +91,7 @@ export async function expectMetricsResult(expectedMetricsResult: Partial<Metrics
 
 export async function expectMetricsJson(expectedMetrics: Partial<Metrics>) {
```

Comment:
I don't think the `expectedMetrics` are used anymore, right? Please remove everywhere. I also think we should rename this to `expectMetricsJsonToMatchSnapshot()`.

Reply:
Right, I have changed this